include/logging/log.h

```diff
@@ -39,6 +39,17 @@ extern "C" {
  */
 #define LOG_ERR(...)    Z_LOG(LOG_LEVEL_ERR, __VA_ARGS__)
 
+/**
+ * @brief Writes an ERROR level message to the log, but only the first @p n times
```

Comment:
English is not my primary language but in zephyr we usual don't use third person verb.
`Write an ERROR ...`

Reply:
I copied that portion of the docstring from the other macros and it was already in third person. However this reminds me to add the "@param n" documentation to the docstring

Reply:
yeah, you're right. This header was one of my first contributions to zephyr and i did that mistake.